circus-train-integration-tests/pom.xml

```diff
@@ -135,12 +148,18 @@
       <groupId>com.hotels</groupId>
       <artifactId>circus-train-common-test</artifactId>
       <version>${project.version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>javax.servlet</groupId>
+          <artifactId>javax.servlet-api</artifactId>
+        </exclusion>
+      </exclusions>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>com.google.code.gson</groupId>
       <artifactId>gson</artifactId>
-      <version>2.5</version>
+      <version>${log4j-api.version}</version>
```

Comment:
I'm pretty sure you didn't mean to do this!

Reply:
:(